helix-term/src/commands/lsp.rs

```diff
@@ -1324,3 +1330,170 @@ pub fn select_references_to_symbol_under_cursor(cx: &mut Context) {
         },
     );
 }
+
+pub fn compute_inlay_hints_for_all_views(editor: &mut Editor, jobs: &mut crate::job::Jobs) {
+    if !editor.config().lsp.display_inlay_hints {
+        for doc in editor.documents.values_mut() {
+            doc.reset_all_inlay_hints();
+        }
+        return;
+    }
```

Comment:
Could we avoid doing this here and instead do the reset once when editor config is updated?

Reply:
Yes, it's better, done